projects/kubernetes/boot.sh

```diff
@@ -15,7 +15,8 @@ set -e
 : ${KUBE_RUN_ARGS:=}
 : ${KUBE_EFI:=}
 : ${KUBE_MAC:=}
-: ${KUBE_PRESERVE_STATE:=}
+: ${KUBE_PRESERVE_STATE:=true}
+: ${KUBE_MASTER_AUTOINIT:=""}
```

Comment:
I think this now cannot be unset by the user either, not sure if this one can be sensibly inverted though, might need to split into two options, one to disable auto init and another to allow the user to provide the options to use for autoinit.

Reply:
i just put the option into the README.